packages/tsr-utils/src/index.ts

```diff
@@ -11,6 +11,14 @@ export type GlobalTsRuntimeObjectKeys = {
   readonly documentation?: string;
 };
 
+export type CanBeReadOnly = {
+  readonly itemsAreReadOnly: boolean;
+} & GlobalTsRuntimeObjectKeys;
+
+export type CanBeOptional = {
```

Comment:
Are you trying to change it to have the optional property directly on the tsRuntimeObject instead of having the `PropertySignature`?

Reply:
I think we should discuss this